Source/Core/VideoCommon/VertexLoaderManager.h

```diff
@@ -76,6 +76,7 @@ extern BitSet8 g_preprocess_vat_dirty;
 extern bool g_bases_dirty;  // Main only
 extern std::array<VertexLoaderBase*, CP_NUM_VAT_REG> g_main_vertex_loaders;
 extern std::array<VertexLoaderBase*, CP_NUM_VAT_REG> g_preprocess_vertex_loaders;
+extern bool g_cp_xf_dirty;
```

Comment:
I don't really like this name, especially since it's only for specific fields in the cpmem/xfmem. Maybe `g_needs_cp_xf_consistency_check` instead?

Reply:
Sounds good.